src/SWP/Bundle/MultiTenancyBundle/SWPMultiTenancyBundle.php

```diff
@@ -27,5 +29,25 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new ConfigurePrefixCandidatesCompilerPass());
         $container->addCompilerPass(new TenantAwareRouterCompilerPass());
         $container->addCompilerPass(new RegisterTenantFactoryCompilerPass());
+        $container->addCompilerPass(new RegisterOrganizationFactoryPass());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSupportedDrivers()
```

Comment:
Are you sure the bundle works with the Doctrine ORM driver? If we're going to support this, tests should probably be written